notebooks/CS_VQE usage.ipynb

```diff
@@ -93,6 +93,14 @@
     "A noncontextual/contextual partition is identied:"
```

Comment:
<p>We should probably mention that the stabilizers are not restricted to the symmetry generators identified above, but can be any product of elements in any such symmetry basis.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/UCL-CCS/symmer/pull/60/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='24'/>